docs/src/howto-guides/025_config_management_plugin.md

```diff
@@ -0,0 +1,129 @@
+
+## Installation of `c8y_configuration_plugin`
+
+To enable the feature, first you need to install the `c8y_configuration_plugin` binary on your device.
+
+### Using the `get-thin-edge_io.sh` script for Debian OS distribution (Recommended)
+
+If your device supports `apt` as a package manager,
+you can install all `thin-edge.io` packages including the `c8y_configuration_plugin` by the `get-thin-edge_io.sh` script.
+If you have already used the `get-thin-edge_io.sh` script,
+this package is installed, by default.
+
+```shell
+curl -fsSL https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/get-thin-edge_io.sh | sudo sh -s
+```
+
+### Using the `c8y_configuration_plugin` Debian package for Debian OS distribution
+
+For Debian OS distribution, we provide the `c8y_configuration_plugin_<version>_<arch>.deb` package as a release asset [here](https://github.com/thin-edge/thin-edge.io/releases).
+
+In case that you didn't use the `get-thin-edge_io.sh` script, you can download the `c8y_configuration_plugin_<version>_<arch>.deb`  package on our [Releases](https://github.com/thin-edge/thin-edge.io/releases) and install it.
+
+```shell
+sudo apt install ./path/to/package/c8y_configuration_plugin_<version>_<arch>.deb
+```
+
+### Extracting from debian package for non-Debian OS distribution
+
+Get the `c8y_configuration_plugin_<version>_<arch>.deb` from our [Releases](https://github.com/thin-edge/thin-edge.io/releases).
+Then, run this command in the directory where the package is stored.
+
+```shell
+ar -x ./c8y_configuration_plugin_<version>_<arch>.deb | tar -xf ./data.tar.xz
+```
+
+The binary is extracted in `<current directory>/usr/bin`.
+For more details, refer to our guide [Extracting from debian package](./015_installation_without_deb_support.md#extracting-binaries-from-deb-packages).
+
+### Building from sources
+
+Follow our guide [Buiding thin-edge.io](./../BUILDING.md) and [Building from source](./015_installation_without_deb_support.md#if-building-from-source).
+
+```shell
+cargo build --release -p c8y_configuration_plugin
+```
+
+A `systemd` unit file for `c8y_configuration_plugin` can be found in the repository at `configuration/init/systemd/c8y-configuration-plugin.service`
+and should be installed on the target in: `/lib/systemd/system/c8y-configuration-plugin.service`.
+
+```shell
+sudo cp <repository_root>/configuration/init/systemd/c8y-configuration-plugin.service /lib/systemd/system/c8y-configuration-plugin.service
+```
+
+## Get started
+
+Before starting anything, make sure [your device is connected to Cumulocity](./../tutorials/connect-c8y.md).
+
+**Step 0**
+Only if you didn't install `c8y_configuration_plugin` by the debian package,
+you need one additional step to initialize the plugin. Run this command.
+
+```shell
+sudo c8y_configuration_plugin --init
+```
+
+**Step 1**
+Open the file `/etc/tedge/c8y/c8y-configuration-plugin.toml` and edit the file.
+
+```toml
+files = [
+    { path = '/etc/tedge/tedge.toml' },
```

Comment:
```suggestion
    { path = '/etc/tedge/tedge.toml', type = 'tedge.toml'},
```

Let's add a `type` for every entry and **NOT** document that it's an optional key for now, as we might have to make it mandatory in an upcoming release to ease the `tedge/configuration_change` MQTT notification mechanism.

Reply:
Sure